### DISCUSSION TITLE
control/install/check_config | Network Error

### DISCUSSION THREAD

**Discussion by felixbay** [AdguardTeam/AdGuardHome#2744] [Q&A]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ ] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * 
* **How did you install AdGuard Home:**
  * 
* **How did you setup DNS configuration:**
  * 
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * 
* **Operating system and version:**
  * 

### Screenshots

<details><summary>Screenshot:</summary>

</details>

**Comment by Aikatsui** [comment]
We don't know how to help you, please follow relevant issue template
https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started

**Comment by crinklywrappr** [comment]
I got this error message when following the instructions on the [docker page](https://hub.docker.com/r/adguard/adguardhome).

Users are likely to overlook the `-d` detatch option when writing their systemd service units.  W/ `Restart=always`, the docker container restarts endlessly and this specific error manifests on the web app.

Suggest adding a bullet point to the docker description so users are less likely to overlook this small detail.
